core/tests/rpc.rs

```diff
@@ -240,7 +240,7 @@ fn test_rpc_subscriptions() {
                     let sent_sender = sent_sender.clone();
                     tokio::spawn(
                         client
-                            .signature_subscribe(sig.clone(), None)
+                            .signature_subscribe(sig.clone(), Some(1))
```

Comment:
This is more restrictive, is it needed?

Reply:
Nope, something I was testing and forgot to revert